vbr/vbrCore/functions/yosh2009.m

```diff
@@ -0,0 +1,66 @@
+function [ VBR ] = yosh2009( VBR )
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  %
+  % [ VBR ] = yosh2009( VBR )
+  %
+  % calculates the conductivity of San Carlos Olivine Aggregate at 10GPa
+  % from temperature and water content
+  %
+  % Parameters:
+  % ----------
+  % VBR    the VBR structure
+  %
+  % Output:
+  % ------
+  % VBR    the VBR structure, with VBR.out.electric.yosh2009_ol structure
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+  % read in eletric parameters
+  ele = VBR.in.electric.yosh2009_ol;
+  T = VBR.in.SV.T; % K (Temperature)
+  Ch2o = VBR.in.SV.Ch2o; % ppm (water content)
+  P = VBR.in.SV.P; % Pa (Pressure)
```

Comment:
Pressure is specified in GPa (I think I told you the wrong thing earlier on...), you'll want to make this change in all the other functions
```suggestion
  P = VBR.in.SV.P_GPa * 1e9; % Pa (Pressure)
```